File: Wei.Web/app/model/sys/WordsSubstitution.js
Ext.define('Wei.model.sys.WordsSubstitution', {
    extend: 'Wei.model.Base',

    fields: [
        { type: 'int', name: 'id', persist: true },
        { type: 'int', name: 'questionbankid', persist: true },
        { type: 'string', name: 'questionbankname', persist: false },
        { type: 'string', name: 'words1', persist: true },
        { type: 'string', name: 'words2', persist: true }

    ]
});

File: Wei.Core/Domain/Sys/WordsSubstitution.cs
using Wei.Core.Domain.Questions;

namespace Wei.Core.Domain.Sys
{
    public class WordsSubstitution : BaseEntity
    {
        /// <summary>
        /// 题卷id
        /// </summary>
        public int? QuestionBankId { get; set; }
        public virtual QuestionBank QuestionBank { get; set; }

        /// <summary>
        /// 匹配词语
        /// </summary>
        public string Words1 { get; set; }

        /// <summary>
        /// 替换词语
        /// </summary>
        public string Words2 { get; set; }

    }
}
